easybuild/easyconfigs/l/libepoxy/libepoxy-1.5.3-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'MesonNinja'
+
+name = 'libepoxy'
+version = '1.5.3'
+
+homepage = 'https://github.com/anholt/libepoxy'
+description = """
+(lib)Epoxy is a library for handling OpenGL function pointer management for you.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_XZ]
+source_urls = [FTPGNOME_SOURCE]
+checksums = ['002958c5528321edd53440235d3c44e71b5b1e09b9177e8daf677450b6c4433d']
+
+builddependencies = [
+    ('Meson', '0.48.1', '-Python-3.6.6', ('foss', '2018b')),
+    ('Ninja', '1.8.2', '', ('foss', '2018b')),
```

Comment:
@boegel mhmm, you mean that it would be more clear to use Meson with GCCcore? I simply took the only available `Meson` [we have](https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/m/Meson).

Obviously, Meson doesn't need all the components of `foss2018b`, so if you'd agree, let's create meson (and Ninja) with some latest-enough GCCcore.. 

Perhaps in context of GTK+3 this is so interesting and general topic that I'll ping you on IRC.

Reply:
oki, it means we need:
- [ ] Python 3.6.6-bare - like [Python-2.7.15-GCCcore-7.3.0-bare](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-2.7.15-GCCcore-7.3.0-bare.eb)
- [ ] Meson for GCCcore
- [ ] Ninja for GCCcore

that's doable.